FirebaseRemoteConfig/Sources/Public/FirebaseRemoteConfig/FIRRemoteConfig.h

```diff
@@ -78,6 +78,16 @@ typedef NS_ERROR_ENUM(FIRRemoteConfigErrorDomain, FIRRemoteConfigError){
     FIRRemoteConfigErrorInternalError = 8003,
 } NS_SWIFT_NAME(RemoteConfigError);
 
+/// Remote Config error domain that handles errors for Realtime.
+extern NSString *const _Nonnull FIRRemoteConfigRealtimeErrorDomain NS_SWIFT_NAME(RemoteConfigRealtimeErrorDomain);
+/// Firebase Remote Config service Realtime error.
+typedef NS_ERROR_ENUM(FIRRemoteConfigRealtimeErrorDomain, FIRRemoteConfigRealtimeError){
+    /// Can't establish config update stream.
+    FIRRemoteConfigRealtimeErrorStream = 8004,
```

Comment:
I think it makes more sense to merge the error with the ones above. There's no need to make seperate domain errors, you can just merge them under RemoteConfigError

Reply:
@ryanwilson

Reply:
I think in our meetings before the API proposal we talked about creating a new error domain for realtime. But I would have no problems moving it into the original RemoteConfigError domain

Reply:
Unfortunately we can't use the same domain when using `NS_ERROR_ENUM` since it uses that to assemble the error enum on the Swift side

Reply:
Sorry I didn't quite realize the errors started at 8004 - there's no reason for this to be the case if we're using a different error enum.

Re: why we can't/ shouldn't add them to `FIRRemoteConfigErrorDomain`, this is an effort to make our errors more locally scoped. It's not possible to get a `FIRRemoteConfigRealtimeErrorStream` error when calling other RC APIs, so that error shouldn't exist in the regular `FIRRemoteConfigError`. 

This is also a limitation imposed by `NS_ERROR_ENUM`, since the error types have a 1 to 1 mapping of domain to Swift error, so that's why we have a second domain. Happy to chat further about it if needed!